src/java.desktop/share/classes/java/awt/Taskbar.java

```diff
@@ -338,6 +338,10 @@ public void setIconImage(final Image image) {
 
     /**
      * Obtains an image of this application's icon.
+     * <p>
```

Comment:
As discussed off-line, preface this with "@apiNote"

Reply:
updated